io.github.diamondburned.gtkcord4.yml

```diff
@@ -0,0 +1,235 @@
+id: io.github.diamondburned.gtkcord4
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: gtkcord4
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --own-name=com.github.diamondburned.gtkcord4
```

Comment:
ideally this would be changed upstream too.

Reply:
I added `--own-name` since upstream uses `com.github` while the flatpak uses `io.github` and it won't run without that line. I contacted the developer about this, and the result of our conversation was to use `xyz.diamondb.gtkcord4`. As soon as the developer merges https://github.com/diamondburned/gtkcord4/pull/102 I will make corresponding changes to the flatpak files.

Reply:
maybe switching to `xyz.diamondb.gtkcord4` is the best here, since that's what upstream is doing.

Reply:
also it's a good opportunity to add the appstream file to upstream as well.

Reply:
https://github.com/diamondburned/gtkcord4/pull/102 was merged and corresponding changes (https://github.com/flathub/flathub/pull/3857/commits/91a12c66443c9a0ef68b507df2ae916cfb33f617) were made. Appstream file added to upstream as well.